dev/PushNotifications/PushNotificationsLongRunningTask/winmain.cpp

```diff
@@ -84,7 +86,7 @@ int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, PSTR /*
         &cookie,
         1));
 
-    InitializePlatform();
```

Comment:
I thought we decided that class factory would own the platform and initialize it atleast once.